### PR TITLE
edit UI review1, review2, implement API review1

### PR DESCRIPTION
**Pull request by VanHung14**

### CODE REVIEW

File: my-app/src/Components/Reviews/ListReviews.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import { useState, useEffect } from "react";
+import { getListReviews } from "../../Api/userApi";
+import { useNavigate } from "react-router-dom";
+import { useLocation } from "react-router-dom";
+import "./ListReviews.css";
+function ListReviews() {
+  const navigate = useNavigate();
+  const search = useLocation().search;
```
Comment (rk-hungdv2) [inline my-app/src/Components/Reviews/ListReviews.jsx]: const search = useLocation().search; → const { search } = useLocation();

File: my-app/src/Components/Reviews/ListReviews.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  const page = new URLSearchParams(search).get("page") || 1;
+  const itemsPerPage = 2;
+  const start = (page - 1) * itemsPerPage;
+  const end = page * itemsPerPage;
+  const [listReviews, setListReviews] = useState([]);
+  const [dataPerPage, setDataPerPage] = useState([]);
+  let [numPages, setNumPages] = useState(1, []);
```
Comment (rk-hungdv2) [inline my-app/src/Components/Reviews/ListReviews.jsx]: let → const

File: my-app/src/Components/Reviews/ListReviews.jsx (hunk 1 above)
Comment (VanHung14) [reply my-app/src/Components/Reviews/ListReviews.jsx]: em fix rồi ạ

File: my-app/src/Components/Reviews/ListReviews.jsx (hunk 2 above)
Comment (VanHung14) [reply my-app/src/Components/Reviews/ListReviews.jsx]: em fix rồi ạ

**Pull request merged**